### ISSUE TITLE
Create API prototype of adding an existing Cloud.gov user to an org

### ISSUE THREAD

**Issue by echappen** [cloud-gov/cg-ui#102]
Blocked by #56 

This prototype will test the following scenario:

- Org Manager can add 1 existing Cloud.gov user to their organization with designated roles ("Existing" user means this user has a Cloud.gov UAA account)
 
The prototype will verify:

- What API calls are necessary for this scenario?
- What kinds of error messages could be returned?
- Approximately how long does this flow take? Is it a noticeable amount of time to the end user?

Build a test page with an email form field, a multi-checklist of roles for this organization, and a submit button that instigates this flow. The page should show an error message on failure and a success message on success. Try out this scenario locally and on a deployed version of the app. 

API endpoints may include:

- viewing all members of an org
- verifying that a new member does not already belong to an org
- adding a member to an org

### Out of scope for this prototype
- How to handle brand new users (i.e. users with no UAA account)  
- UI styling is also out of scope—prototype will look ugly
- Emailing users to notify them of their new org
- UI testing - let's focus on basic testing for the API requests instead of for UI elements, since the UI will be replaced soon

### Acceptance Criteria

- [x] on a test page, org Manager can submit an email for a new member to be added to an org
- [x] Added member persists in the list of org members
- [x] Error message is shown on failure
- [x] Success message is shown on success
